content/applications/general/integrations/translate.rst

```diff
@@ -0,0 +1,118 @@
+================
+
+*Google Translate* can be used to translate user generated text in the Odoo chatter. To translate a
+user's text from another language click the *three dot* menu to the right of the chatter. Then
+select :guilabel:`Translate`. The language will translate to the *language* set on the user's
+preferences.
+
+.. seealso::
+   :ref:`language/change-user-language`
+
+.. image:: translate/google-translate.png
+   :align: center
+   :alt: Google Translate present in an Odoo database's chatter.
+
+Integration is simple with some configuration on *Google's Developer Console* and copy and pasting
+an :guilabel:`API key` into Odoo.
+
+Google API Console
+==================
+
+A majority of the setup for integrating *Google Translate* into Odoo is done with *Google*. Once the
+following processes are complete an :guilabel:`API key` will be created to input in Odoo.
+
+.. seealso::
+   `Google Translate setup on Google <https://cloud.google.com/translate/docs/setup>`_
+
+Create a new project
+--------------------
+
+To get started, go to the `Google API Console <https://console.developers.google.com>`_. Log in with
+the *Google Workspace* account if there is one, otherwise log in with a personal Gmail account (this
+should match the email address that has billing attached to it).
+
+After that, click on :guilabel:`Create Project`, located on the far right of the :guilabel:`OAuth
+consent screen`.
+
+.. tip::
+   If the *Google Developer Console* has existing projects, click the drop-down menu next to the
+   :guilabel:`Google Cloud` icon and a popover will appear. Next, click :guilabel:`New Project` in
+   the top right of the popover.
+
+On the :menuselection:`New Project` screen, rename the :guilabel:`Project name` to `Odoo Translate`
+and browse for the :guilabel:`Location`. Set the :guilabel:`Location` as the *Google Workspace
+organization*. If a personal Gmail account is being used, then leave the :guilabel:`Location` as
+:guilabel:`No Organization`.
+
+.. image:: translate/new-project.png
+   :align: center
+   :alt: Project Name and Location for Google OAuth.
+
+Click on :guilabel:`Create` to finish this step.
+
+API Library
+-----------
+
+Next the :guilabel:`Cloud Translation API` needs to be installed on this newly created project.
+First click :guilabel:`Library` in the left menu. The search the term `Cloud Translation API` and
+click into the result. This should be a *Google Enterprise API* labeled `Cloud Translation API`.
+Click :guilabel:`Enable` to install the library on this project.
+
+.. important::
+   Using the *Google Translate* API requires a current billing account with `Google
+   <https://https://myaccount.google.com/>`_.
+
+Once a billing account is setup with *Google* and the library is enabled, click :guilabel:`Manage`
+to finish configuration on the API.
+
+Create Credentials
+------------------
+
+Now that the project is set up and library is enabled, credentials should be created. These include
```

Comment:
```suggestion
Now that the project is set up and the library is enabled, credentials need to be created. These include
```
Changing this to "need" unless it is an optional step, in which case I would add a note admonition block to make that clear